src/js/components/Navigation/HeaderBar.jsx

```diff
@@ -443,11 +444,10 @@ class HeaderBar extends Component {
   closeShareModal () {
     AppObservableStore.setShowShareModal(false);
     AppObservableStore.setShareModalStep('');
-    const pathname = normalizedHref();
-    // console.log('HeaderBar closeShareModal pathname:', pathname);
-    if (stringContains('/modal/share', pathname) && isWebApp()) {
-      const pathnameWithoutModalShare = pathname.replace('/modal/share', '');  // Cordova
-      // console.log('Navigation closeShareModal pathnameWithoutModalShare:', pathnameWithoutModalShare);
+    const { location: { href } } = window;
```

Comment:
I changed this code to fix a bug with the modals, and this change is going back to what was breaking.

Reply:
I have no idea why I would have changed that, it must have been a manual merge error.  I'll remove the change in the next PR.